src/patterns/CommonTable/components/TemporaryOperationArea.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Cluster, FaCloudDownloadAltIcon, FaSearchIcon, FilterDropdown, Input, Pagination } from 'smarthr-ui'
+import styled, { css } from 'styled-components'
+import { PageCounter } from './PageCounter'
+
+type Props = {
+  searchValue: string
+  pagination: { totalPages: number; currentPage: number; totalCount: number; limitValue: number }
+}
+
+export const TemporaryOperationArea: React.FC<Props> = (props) => {
+  const {
+    searchValue,
+    pagination: { totalPages, currentPage, totalCount, limitValue },
+  } = props
+
+  return (
+    <HeaderCluster justify="space-between" align="center">
+      <form role="search" onSubmit={(e) => e.preventDefault()}>
+        <Cluster align="center" gap={1}>
+          <Cluster gap={0.5}>
+            <label>
+              <Input
+                prefix={<FaSearchIcon size={14} alt="検索" color="TEXT_GREY" />}
```

Comment:
デフォルト `1em` になるので不要です!

```suggestion
                prefix={<FaSearchIcon alt="検索" color="TEXT_GREY" />}
```

Reply:
消しました！